src/kata2.booleanToWord.js

```diff
@@ -1,3 +1,9 @@
-const booleanToWord = (boolean) => {};
+const booleanToWord = (boolean) => {
```

Comment:
The logic here works perfectly, but it could be refactored into one line :)

The first thing is to use === instead of == as default.
You can change the if statement to use the ternary operator:
````
const booleanToWord = (boolean) => (boolean ? "Yes" : "No");
`````